### ISSUE TITLE
Compression not working

### ISSUE THREAD

**Issue by huegelc** [masc3d/btrfs-sxbackup#46]
Hi,

lzop compression doesnt seem to work

```
btrfs-sxbackup v0.6.1

```

```
 Command ['ssh', '-o', 'ServerAliveInterval=5', '-o', 'ServerAliveCountMax=3', 'root@176.9.106.118', 'ionice -c3 btrfs send "/var/lib/lxd/containers/test/.sxbackup/.temp.8dc8849378ee4b0198d142d2b074980d" | lzop -1'] failed with error code 127 (bash: lzop: command not found
```

```
which lzop
/usr/bin/lzop
```

**Comment by masc3d** [comment]
`lzop` is apparently not available on your destination system's (`176.9.106.118`) path.

**Comment by huegelc** [comment]
You´re absolutely right. I checked only the source. Thank you
